src/qtism/runtime/common/State.php

```diff
@@ -85,7 +85,8 @@ public function getVariable($variableIdentifier): ?Variable
      *
      * @return VariableCollection A collection of Variable objects.
      */
-    public function getAllVariables(): VariableCollection
+    #[\ReturnTypeWillChange]
```

Comment:
Why not just the return type here?
If that breaks test cases on deliver be, it's the test cases that should be adjusted instead.

Reply:
Attempted to resolve that [here](https://github.com/oat-sa/qti-sdk/pull/346/files/d2585cd856ee70afe05e394fdff4819ca8f39e9a..81869543691aa149285d7c50f90acdf74b273594).